src/common/reducers/entities/snap.js

```diff
@@ -10,32 +10,32 @@ export default function snap(state={}, action) {
   };
 
   switch(action.type) {
-    case RegisterNameActionTypes.CHECK_NAME_OWNERSHIP_REQUEST:
-      return {
-        ...state,
-        snapcraftData: {
-          ...state.snapcraftData,
-          isFetching: true
-        }
-      };
-    case RegisterNameActionTypes.CHECK_NAME_OWNERSHIP_SUCCESS:
-      return {
-        ...state,
-        snapcraftData: {
-          ...state.snapcraftData,
-          isFetching: false,
-          nameOwnershipStatus: action.payload.status
-        }
-      };
-    case RegisterNameActionTypes.CHECK_NAME_OWNERSHIP_ERROR:
-      return {
-        ...state,
-        snapcraftData: {
-          ...state.snapcraftData,
-          isFetching: false,
-          nameOwnershipStatus: null
-        }
-      };
+    // case RegisterNameActionTypes.CHECK_NAME_OWNERSHIP_REQUEST:
+    //   return {
+    //     ...state,
+    //     snapcraftData: {
+    //       ...state.snapcraftData,
+    //       isFetching: true
+    //     }
+    //   };
+    // case RegisterNameActionTypes.CHECK_NAME_OWNERSHIP_SUCCESS:
+    //   return {
+    //     ...state,
+    //     snapcraftData: {
+    //       ...state.snapcraftData,
+    //       isFetching: false,
+    //       nameOwnershipStatus: action.payload.status
+    //     }
+    //   };
+    // case RegisterNameActionTypes.CHECK_NAME_OWNERSHIP_ERROR:
+    //   return {
+    //     ...state,
+    //     snapcraftData: {
+    //       ...state.snapcraftData,
+    //       isFetching: false,
+    //       nameOwnershipStatus: null
+    //     }
+    //   };
```

Comment:
Ah... Rushed up with a commit, will remove of course as that's the part that was moved.